### PR TITLE
PACT-732 Added updateEditSetRecord to the request channel

### PR DESCRIPTION
**Pull request by userjd**

### CODE REVIEW

File: schema/omega-services.async.api.yaml [hunk 1]
```diff
@@ -59,7 +60,7 @@ channels:
           - $ref: '#/components/messages/editSet'
           - $ref: '#/components/messages/editSetSummary'
           - $ref: '#/components/messages/editSetRecord'
-          - $ref: '#/components/messages/updateEditSetRecord'
+          - $ref: '#/components/messages/editSetRecordUpdate'
```
Comment (rwalpole) [inline schema/omega-services.async.api.yaml:63]: We can remove this from here. When the user clicks "Save changes" they are always taken to the confirmation page and don't get the whole record back unless they return to the edit record page, in which case they would make a fresh call to `getEditSetRecord` .

**Review by rwalpole — CHANGES_REQUESTED** [review]

File: schema/omega-services.async.api.yaml (hunk 1 above)
Comment (userjd) [reply schema/omega-services.async.api.yaml:63]: @rwalpole in that case, the `editSetRecordUpdate` message seems redundant as this is not being used and can be removed too?

Comment (userjd) [reply schema/omega-services.async.api.yaml]: > We can remove this from here. When the user clicks "Save changes" they are always taken to the confirmation page and don't get the whole record back unless they return to the edit record page, in which case they would make a fresh call to `getEditSetRecord` .

Removed.

Comment (rwalpole) [reply schema/omega-services.async.api.yaml]: Agreed @userjd . I think there are some other inconsistencies in here as well which I have raised in https://national-archives.atlassian.net/browse/PACT-789.

**Review by rwalpole — APPROVED** [review]

**Pull request merged**